packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -345,6 +345,10 @@ export const EVENT_TRIGGERS = {
   MEDIA_UPDATE: 'media:update',
   MEETING_STARTED_SHARING_LOCAL: 'meeting:startedSharingLocal',
   MEETING_STOPPED_SHARING_LOCAL: 'meeting:stoppedSharingLocal',
+  MEETING_LOCAL_MEDIA_STARTED: 'meeting:media:local:start',
```

Comment:
Nitpick but could you rename the consts to match the string they are defining? 
```suggestion
  MEETING_MEDIA_LOCAL_STARTED: 'meeting:media:local:start',
```